tests/exchange/test_okex.py

```diff
@@ -0,0 +1,166 @@
+from unittest.mock import MagicMock  # , PropertyMock
+
+from tests.conftest import get_patched_exchange
+
+
+def test_get_maintenance_ratio_and_amt_okex(
+    default_conf,
+    mocker,
+):
+    api_mock = MagicMock()
+    default_conf['trading_mode'] = 'futures'
+    default_conf['margin_mode'] = 'isolated'
+    default_conf['dry_run'] = True
+    api_mock.fetch_leverage_tiers = MagicMock(return_value={
+        'SHIB/USDT:USDT': [
+            {
```

Comment:
Here is another instance where I'm testing expected inputs and outputs, but with the Okex api response (Okex doesn't have the maintenance amount in info)

Reply:
Do we need "info"? 
in my experience, "info" is the raw exchange response - which i'd assume ccxt is abstracting away.
or is some field missing in the unification (i was assuming this is a new endpoint you added, so should have everything we need).

Reply:
The binance response contains `cum` in info, which is the maintenance amount, which is needed in the binance liquidation price formula, the way we we're calculating it before gave the wrong numbers, and I thought it was easier just to use the number that the exchange returns. This here is just testing that `maintAmt` gets set to `None` with the Okex response. 

The reason `cum` is only in info is that `binance` is the only exchange that returns it, so it doesn't make sense to put it in the unified response, but it is required for the binance liquidation price formula